docs/src/content/docs/es/challenges/angular/5-crud.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 🟢 Aplicación CRUD
+description: El desafío 5 se trata de refactorizar una aplicación CRUD.
+author: thomas-laforge
+challengeNumber: 5
+command: angular-crud
+sidebar:
+  order: 2
+---
+
+## Information
```

Comment:
Looks like we missed this title, can we add a translation here as well?

Reply:
Thanks Nelson, let me fix it.

Reply:
Fixed